includes/Core/CLI/CLI_Command.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Site Kit CLI Command
+ *
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\CLI;
+
+use Google\Site_Kit\Context;
+use WP_CLI_Command;
+
+/**
+ * Base CLI Command class.
+ *
+ * @since n.e.x.t
```

Comment:
See previous comment about common annotations that all classes should have (would be nice if we had code sniffer rules for this).

Reply:
Updated.

Reply:
Updated.